pages/record/_.vue

```diff
@@ -131,17 +132,30 @@
     },
     data() {
       return {
+        agents: null,
         altTitle: null,
+        concepts: null,
         description: null,
         error: null,
+        fields: null,
         identifier: null,
         image: null,
-        fields: null,
         media: null,
+        relatedEntities: null,
         title: null
       };
     },
     computed: {
+      europeanaAgents() {
+        return (this.agents || []).filter((agent) => agent.about.startsWith('http://data.europeana.eu/agent/'));
+      },
+      europeanaConcepts() {
+        return (this.concepts || []).filter((concept) => concept.about.startsWith('http://data.europeana.eu/concept/'));
+      },
```

Comment:
Could these ever start with `https://`?

Reply:
Certainly not in the foreseeable future, and if they did, it would be a transitioned change we would know about well in advance and could account for then.